apis/v1alpha1/validation_test.go

```diff
@@ -38,7 +38,7 @@ func TestValidateResouceMapping(t *testing.T) {
 					Name:     "//pubsub.googleapis.com/projects/test-project/topics/test-topic",
 				},
 				Contacts: &Contacts{
-					Email: []string{"pmap.gmail.com"},
+					Email: []string{"pmap.example.com"},
```

Comment:
change to @

Reply:
I think this is the invalid email test, we want an invalid email here?

Reply:
Sorry, I missed that(the previous code lines were hidden by default). Maybe change it to `invalid.example.com` to make it more obvious

Reply:
Done.